R/peaks-functions.R

```diff
@@ -224,6 +224,20 @@ NULL
 #'   between `x` and `y` are returned by `type = "inner"`, i.e. only
 #'   peaks present in both spectra are reported.
 #'
+#' - `joinPeaksGnps` matches/maps peaks between spectra with an approach
+#'   similar to the one used in GNPS: peaks are considered matching if a) the
```

Comment:
Could you clarify what you mean by *similar*? Same algorithm but different implementation? Any tests that assess if identical or what the deviations are?

Reply:
It's actually the same approach but different implementation (GNPS is based on python). There are differences, but after the 3rd decimal.

Reply:
May be simply mention this in the documentation.

Reply:
Done